### PR TITLE
[W5.7][F10-3]Liau Jian Jie

### PR DESCRIPTION
**Pull request by liaujianjie**
## Description

Adds a `sort` command to sort the contacts by name in ascending alphabetical order.

## Usage

`sort` (no parameters)

### CODE REVIEW

**Comment by liaujianjie** [comment]
Work in progress

**Comment by liaujianjie** [comment]
Ready for review

**Comment by dongsiji** [comment]
Good job adding the new command sort, maybe can include more test cases testing on corner case for your code.

File: src/seedu/addressbook/data/AddressBook.java [hunk 1]
```diff
@@ -72,6 +79,6 @@ public UniquePersonList getAllPersons() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddressBook // instanceof handles nulls
-                        && this.allPersons.equals(((AddressBook) other).allPersons));
+                && this.allPersons.equals(((AddressBook) other).allPersons));
```
Comment (dalessr) [inline src/seedu/addressbook/data/AddressBook.java:82]: This change violates the coding standard (are `||` and `&&` at the same level here?)

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```
Comment (dalessr) [inline src/seedu/addressbook/data/person/UniquePersonList.java:3]: Coding standard violation here. Should specify all classes used in this file

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (dalessr) [inline src/seedu/addressbook/parser/Parser.java:14]: Same as above. (I know this is actually auto-changed by IDE. Thus need to be cautious & check the changes every time before committing :-p)

**Review by dalessr — APPROVED** [review]
@liaujianjie Well done for the `sort` command 👍

Please close the PR after looking at the comments below.
